libs/portals/core/src/types/portalCore.ts

```diff
@@ -57,6 +57,21 @@ export interface PortalNavigationItem {
    * Active state if current path is exact match
    */
   activeIfExact?: boolean
+
+  /**
+   * Service provider
+   */
+  serviceProvider?: string
+
+  /**
+   * Parent
```

Comment:
~Redundant comment as it repeats the variable name. Would be nice to have a short description of the use case when `parent` is used.

Reply:
I believe this was added when the initial work was started but is not being used now.